android/kotlin/AmpliApp/app/src/main/java/com/amplitude/ampli/Ampli.kt

```diff
@@ -77,10 +77,13 @@ class Identify private constructor(
     constructor(
         requiredNumber: Double,
         optionalArray: Array<String>? = null
-    ) : this(mapOf(
-        *(if (optionalArray != null) arrayOf("optionalArray" to optionalArray) else arrayOf()),
-        "requiredNumber" to requiredNumber
-    ))
+    ) : this(
+        eventProperties = mapOf(
+            *(if (optionalArray != null) arrayOf("optionalArray" to optionalArray) else arrayOf()),
+            "requiredNumber" to requiredNumber
+        ) as Map<String, Any?>?,
+        options = null as EventOptions?
```

Comment:
Do we need this options null here?

Reply:
+1 https://github.com/amplitude/javascript/pull/14823#discussion_r809263556

Reply:
Fixed